code/modules/admin/verbs/randomverbs.dm

```diff
@@ -704,6 +704,11 @@ Traitors and the like can also be revived with the previous role mostly intact.
 		if(istype(O,/turf))
 			var/turf/T=O
 			T.ChangeTurf(T.get_underlying_turf())
+		if(istype(O,/mob/living/carbon/human))
```

Comment:
please don't play this sound when there is no ckey in the mob or the ckey is the admin's own

the sound file is very annoying and this will annoy me when I'm thunderdome testing

Reply:
oh yeah fair enough — do you know what the code would be for detecting a soul in a mob?

Reply:
Also, the gravitational singulo drink (#29125) checked for mind inside the mob (so you could not singuloose via the drink without a player sacrifice), perhaps that helps.

Reply:
> Also, the gravitational singulo drink (#29125) checked for mind inside the mob (so you could not singuloose via the drink without a player sacrifice), perhaps that helps.

thank you very much!